qcodes/instrument_drivers/signal_hound/USB_SA124B.py

```diff
@@ -166,28 +168,28 @@ def openDevice(self):
         self.get('device_type')
 
     def closeDevice(self):
-        self.log.info('Closing Device with handle num: ',
-                      self.deviceHandle.value)
+        log.info('Closing Device with handle num: '
+                 f'self.deviceHandle.value')
```

Comment:
shouldn't it be `f'{self.deviceHandle.value}'` (with `{` and `}`)?

Reply:
Yes. Good one.